docker-compose.yml

```diff
@@ -17,12 +17,12 @@ services:
     network_mode: bridge
     ports:
       - 5000:5000
-  
+
   uta:
     # Test:
-    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
-    # 249909
-    image: biocommons/uta:uta_20180821
+    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from uta_20210129b.transcript'
```

Comment:
I think the corresponding documentation on the README.md should be changed as well (this took me a minute when I was trying to use the install instructions to figure out.

```
snafu$ psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
249909
```

Reply:
i guess I missed the review window ;)

Reply:
Good catch!!